backend/mora/service/address_handler/phone.py

```diff
@@ -66,9 +68,8 @@ def get_lora_properties(self):
     def _get_mo_properties(self):
         properties = super()._get_mo_properties()
         if self.visibility:
+            c = lora.Connector()
             properties.update({
-                mapping.VISIBILITY: {
-                    'uuid': self.visibility
-                }
+                mapping.VISIBILITY: facet.get_one_class(c, self.visibility)
```

Comment:
Should we cache these?

Reply:
Caching classes is some of the most low hanging fruit as they bound to almost never change, so we should definitely do that, but I am not really sure how to do it yet.

I'll add a ticket for it, (if there isn't one already) and then we can talk about the design for it at some point.